<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DashBoard</class>
 <widget class="QMainWindow" name="DashBoard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
color:rgb(0, 0, 0)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Images/uniandes.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Canvas" name="canvas" native="true">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>130</y>
      <width>351</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2 solid;
border-color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start_simulation">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>390</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Simular</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdb_lineal">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>99</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Lineal</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdb_trian">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>99</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Triangular</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdb_penta">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>99</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Pentogonal</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graph_canvas">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>351</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="formation_label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>160</width>
      <height>160</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Images/vertical_line.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="restart_simulation">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>440</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Reiniciar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Canvas</class>
   <extends>QWidget</extends>
   <header>canvas.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
